src/classes/HouseholdNamingService.cls

```diff
@@ -39,6 +39,39 @@ public without sharing class HouseholdNamingService {
 
     private HouseholdSettings settings = new HouseholdSettings();
 
+    @TestVisible
```

Comment:
Some future planning here, non-blocking.

Can we eventually have dependency injection without using `@TestVisible`? These dependencies could be passed into the class via a constructor, but some other class would need to manage that - and I think that'd be an [Application class](https://github.com/apex-enterprise-patterns/fflib-apex-common/blob/master/sfdx-source/apex-common/main/classes/fflib_Application.cls). When `HouseholdNamingService` would need an instance of `ContactSelector`, it'd grab whatever one is given to it by the Application Class. Does that seem viable in the long-run?

Reply:
Yep, exactly! Once we have a framework like [Force-DI](https://github.com/apex-enterprise-patterns/force-di) in place, then the injection of these class member collaborators become one-liners `(forceDI.getInstance(Collaborator.class))`, and the Application is configured for run-time injection using custom metadata types.  At that point, the domain can be configured at run time in a test by telling the Application to build an instance using mock collaborators instead of real ones, etc.